backend/app/models/metadata.py

```diff
@@ -206,6 +222,12 @@ def definition_is_valid(cls, v):
     class Settings:
         name = "metadata"
 
+    class Config:
+        # Serialization Config Options
+        # Specify JSON key names
+        # This will rename the field `context` to `@context` in the JSON output
+        fields = {"context": "@context"}
```

Comment:
Just curious, no need to actually say "alias"?

Reply:
Good catch Chen. This is an extra piece of code block. I had the Config class set up above. I removed this piece of code block in my next commit.